edk2toolext/invocables/edk2_setup.py

```diff
@@ -113,8 +132,8 @@ def Go(self):
                 # Clean any submodule repos.
                 if required_repos:
```

Comment:
maybe these variables should be renamed (vs code makes this really easy with refactor > rename or f2). required_submodule instead of required_repo

Reply:
agree